### ISSUE TITLE
Documentation about running node in a container

### ISSUE THREAD

**Issue by mydatascience** [TRON-US/go-btfs#633]
https://docs.btfs.io/docs/run-a-btfs-host 
doesn't describe how to run a  host in a container. 

/hostui gives 404 on the case of exposing 8080

I was able to see host using 5001 but it looks
![Screen Shot 2020-08-11 at 9 13 56 PM](https://user-images.githubusercontent.com/3028020/89933474-c4a92780-dc17-11ea-93a5-48fe00dd8209.png)

not valid

Update Readme for docker, please

**Comment by taiyangc** [comment]
Good catch. If you have free time feel free to submit a PR. Otherwise, it may take some time for us to update it. Thanks for your patience.
